src/lib/data/aggregates.ts

```diff
@@ -0,0 +1,40 @@
+import {db, Q, RideModel} from '../../database';
+
+export type RideAggregate = {
+  avg_speed: number;
+  avg_power: number;
+  avg_cadence: number;
+  avg_hr: number;
+  max_speed: number;
+  max_power: number;
+  max_cadence: number;
+  max_altitude: number;
+  max_hr: number;
+  min_hr: number;
+};
+
+export async function getRideAggregates(
+  ride: RideModel,
+): Promise<RideAggregate> {
+  // TODO: figure out how to test this func.
```

Comment:
Is this todo, still valid since you have the mock for getRideAggregates?

Reply:
Yeah, I guess ideally we wouldn't mock any core functionality and only mock third-party stuff. So I was hoping there is some way to run sqlite in the test env with watermelon.

Reply:
from reading the docs it didn't seem like it was possible or very easy :/